packages/react/src/themes/font-awesome/components/Icon/iconStyles.ts

```diff
@@ -0,0 +1,11 @@
+import { ComponentSlotStylesInput, ICSSInJSStyle } from '../../../types'
+import { IconProps } from '../../../../components/Icon/Icon'
+import { IconVariables } from '../../../teams/components/Icon/iconVariables'
+
+const iconStyles: ComponentSlotStylesInput<IconProps, IconVariables> = {
+  fontRoot: (): ICSSInJSStyle => ({
+    fontWeight: 900, // required for the fontAwesome to render
```

Comment:
It's great that the only required override is actually related to the font awesome icons. I would really like the same thing to be applicable for svgs :)